applications/ShapeOptimizationApplication/python_scripts/model_part_controller_factory.py

```diff
@@ -42,6 +42,10 @@ def __init__(self, model_settings, model):
             },
             "mesh_motion" : {
                 "apply_mesh_solver" : false
+            },
+            "output_settings": {
```

Comment:
`output` is not the best term here since it might be confused with settings of the optimization output. In the end we do all the evaluation in this folder. 

Maybe the terms like `working_directory` or `iteration_scope` or `working_scope` fit better. 
I did not come up if something better either

Reply:
done :)